frontend/src/v5/helpers/viewpoint.helpers.ts

```diff
@@ -20,11 +20,8 @@ import { generateViewpoint as generateViewpointV4, getNodesIdsFromSharedIds, toS
 import { formatMessage } from '@/v5/services/intl';
```

Comment:
In custom ticket groups, editing or creating a new group deselects all the other ones